pkg/util/detector/detector.go

```diff
@@ -221,13 +227,22 @@ func (d *ResourceDetector) EventFilter(obj interface{}) bool {
 		strings.HasPrefix(clusterWideKey.Namespace, names.KarmadaReservedNSPrefix) {
 		return false
 	}
-
 	if clusterWideKey.Group == clusterv1alpha1.GroupName ||
 		clusterWideKey.Group == policyv1alpha1.GroupName ||
 		clusterWideKey.Group == workv1alpha1.GroupName {
 		return false
 	}
 
+	if names.IgnoreKinds.Has(clusterWideKey.Kind) {
+		return false
+	}
+	watchedKinds := strings.Split(d.WatchedKinds, ",")
+	if len(watchedKinds) > 0 {
+		kindsSet := sets.NewString(watchedKinds...)
+		if kindsSet.Has(clusterWideKey.Kind) {
+			return true
+		}
+	}
 	return true
```

Comment:
mathed resource kind will return true with line 239-245, here return false?

Reply:
`IgnoreKinds ` such as Node and Event will ignored by default, if watchedKinds is set, then detector will only filter these kinds events.